### PR TITLE
Sam/Stress Test Fixes

### PR DESCRIPTION
**Pull request by samholmes**
This fixes a bug after switching to V2 API. It fixes incorrect response handling, and some cleanup remove unnecessary cleaners.

### CODE REVIEW

File: src/bin/stress-test/types.ts [hunk 1]
```diff
@@ -5,19 +5,34 @@ import {
...
   asString,
-  asUnknown,
   asValue
 } from 'cleaners'
 
+export interface ErrorObj extends Error {
+  [key: string]: any
+}
+export const asErrorObj = (raw: any): ErrorObj => {
+  const clean = asObject({
```
Comment (swansontec) [inline src/bin/stress-test/types.ts:15]: Once you've validated that this is an Error object, why not return it as-is? This copying seems unnecessary.

File: src/bin/stress-test/utils/utils.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ export const criticalError = (err: any): void => {
 
 // Error handlers
 
-export const isAcceptableError = (err: any): boolean =>
+export const isErrorWorthRetry = (err: any): boolean =>
```
Comment (swansontec) [inline src/bin/stress-test/utils/utils.ts:78]: Ah, this name makes much more sense. 👍

**Review by swansontec — APPROVED** [review]
Approved, pending one question.

File: src/bin/stress-test/types.ts (hunk 1 above)
Comment (samholmes) [reply src/bin/stress-test/types.ts:15]: The error needs to be serializable through IPC. Node's IPC API returns `{}` for the error object IIRC.

**Review by swansontec — APPROVED** [review]
Oh, derp. That makes sense - it's `.ts` now, but it doesn't stay `.ts` once Sucrase is done.

File: src/bin/stress-test/types.ts (hunk 1 above)
Comment (swansontec) [reply src/bin/stress-test/types.ts:15]: Ok, that makes perfect sense then.

**Pull request merged**